lib/ruleHelper.js

```diff
@@ -111,7 +111,7 @@ RuleHelper.prototype = {
             break;
         case "MemberExpression":
             methodName = node.property.name;
-            objectName = node.object.name;
+            objectName = node.object.name || this.context.getSource(node.object);
```

Comment:
Right, this would disallow `doSomething(aDocument).write()`.
Maybe that's not OK.
I am not sure....Hm!